voluncheer/profiles/tests/test_views.py

```diff
@@ -108,3 +110,27 @@ def test_confirm_attendance(self):
         confirm_attendance(get_request, self.opp.pk)
         self.opp.refresh_from_db()
         self.assertEqual(self.opp.attended_volunteers.count(), 0)
+
+
+class GalleryTest(TestCase):
+    """Test cases for Gallery based functions"""
+
+    def test_create_post(self):
```

Comment:
Please add docstrings to explain the purpose of each test.

Reply:
Done